### PR TITLE
Emag a spesspod for !!FUN!!

### PR DESCRIPTION
**Pull request by monster860**
This PR makes it so that if you emag the spacepod, the safeties are disabled, and that means running into shit will do some damage to the things you run into (Either by ex_act, or by stun+brute if it's a mob), and also do some damage to the spacepod. Now, an emagged spacepod can be used to bust into someone's office and take some shit.

:cl:
rscadd: Emagging a spacepod will make it do damage when running into things
/:cl:

### CODE REVIEW

**Comment by KasparoVy** [comment]
:+1:, pretty nifty feature

**Comment by Fox-McCloud** [comment]
:-1:

**Comment by Spacemanspark** [comment]
:+1: this will be fun.

**Comment by johannhawk** [comment]
RAMMING SPEED

**Comment by Aurorablade** [comment]
Do not want bad this is coming from a space pod fan

**Comment by DaveTheHeadcrab** [comment]
Oh lord why.

File: code/game/vehicles/spacepods/spacepod.dm [hunk 1]
```diff
@@ -240,6 +243,9 @@
 			processing_objects.Add(src)
 
 /obj/spacepod/attackby(obj/item/W as obj, mob/user as mob, params)
+	if(istype(W, /obj/item/weapon/card/emag) || istype(W, /obj/item/weapon/melee/energy/blade))
```
Comment (Regen1) [inline code/game/vehicles/spacepods/spacepod.dm]: NINJACODE IS COMING BACK

*SCREAM

**Comment by TheDZD** [comment]
Honestly I don't really like the idea of this either. Spacepods running through stunning at smacking people after being emagged is particularly great, especially considering the most likely person to make use of this would be the mechanic, who would just be able to unlock lockboxed spacepod parts anyways.

**Comment by monster860** [comment]
Keep in mind the spacepod takes damage too.

**Comment by Kevak** [comment]
I'd say if you could emag repeatedly to toggle it on and off that would be good. But just making a spacepod extremely easy to damage and only good for looting is bad. (Also spacepods take absolutely forever to repair.)

**Comment by Necaladun** [comment]
Would have to know more about the numbers of how it could work before I could approve this. If I sneak it into escape, how many people could I run down with it and kill before the pod would explode?
How many windows, walls, reinforced walls could i crash through, if I could at all?

**Comment by Fox-McCloud** [comment]
You could bump into something 50 times before your pod blows.

File: code/game/vehicles/spacepods/spacepod.dm [hunk 2]
```diff
@@ -315,6 +321,10 @@
 		else
 			user << "\blue <b>\The [src] is fully repaired!</b>"
 
+/obj/spacepod/emag_act(mob/user as mob)
+	if(!emagged)
+		emagged = 1
+		user << "<span class='notice'>You disable the safeties on \the [src]</span>"
```
Comment (tigercat2000) [inline code/game/vehicles/spacepods/spacepod.dm]: to_chat

**Comment by Fox-McCloud** [comment]
closing due to strong disapproval amongst the codertrain and lack of updates/changes to the PR.

**Pull request closed without merging**